dbt_subprojects/tokens/models/tokens/abstract/tokens_abstract_erc20.sql

```diff
@@ -16,4 +16,7 @@ FROM (VALUES
     , (0x84a71ccd554cc1b02749b35d22f684cc8ec987e1, 'USDC.e', 6)
     , (0x9ebe3a824ca958e4b3da772d2065518f009cba62, 'PENGU', 18)
     , (0x000000000000000000000000000000000000800a, 'ETH', 18)
-) AS temp_table (contract_address, symbol, decimals) 
\ No newline at end of file
+    , (0xc325b7e2736a5202bd860f5974d0aa375e57ede5, 'ABSTER', 18)
+    , (0x85ca16fd0e81659e0b8be337294149e722528731, 'NOOT', 18)
+    , (0xd045e0686a784e272e651fc2c08324edabe7403a, 'CHENGU', 18)
```

Comment:
```suggestion
```

Reply:
this one already exists from automated source
```
select *
from tokens.erc20
where blockchain = 'abstract'
and contract_address = 0xd045e0686a784e272e651fc2c08324edabe7403a
limit 100
```

Reply:
we may also see the other two appear shortly, if they are recently deployed onchain and automated dataset recognizes them